pkgs/development/tools/misc/stlink/default.nix

```diff
@@ -55,8 +48,8 @@ in stdenv.mkDerivation rec {
   meta = with lib; {
     description = "In-circuit debug and programming for ST-Link devices";
     license = licenses.bsd3;
-    # upstream says windows, linux/unix but dropped support for macOS in 1.8.0
-    platforms = platforms.linux; # not sure how to express unix without darwin
+    platforms = platforms.unix;
+    badPlatforms = platforms.darwin;
```

Comment:
The change is "express unix without darwin" and does not re-allow darwin.

Reply:
`platform.unix` includes `darwin` which upstream explicitly states is no longer supported...

Reply:
See https://nixos.org/manual/nixpkgs/unstable/#var-meta-badPlatforms.

Reply:
nice, thanks!